app/services/auto-approval/auto-approval-process.js

```diff
@@ -52,7 +57,13 @@ module.exports = function (claimData) {
             return result
           })
       } else {
-        return result
+        var insertClaimEventDataCalls = []
```

Comment:
combine to single event, use `<br/>` to break up lines and display in internal using {{ event['note']|safe }} (only for internal comments NOT user comments)